assignments/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/assign.expiresaftertag.json

```diff
@@ -0,0 +1,26 @@
+{
+    "properties": {
+      "metadata": {
+        "assignedBy": "PlatOps"
+      },
+      "description": "Policy to check required tag",
+      "displayName": "HMCTS Expires After Tagging - CFT SBOX (DCD-CFTAPPS-SBOX)",
```

Comment:
what's the reason for tagging this at subscription level instead of management group?

Reply:
Was to original test the policy.  Should I remove it now as it will be applying from the management group level?